crates/cheatcodes/src/inspector.rs

```diff
@@ -69,7 +68,7 @@ use revm::{
 };
 use serde_json::Value;
 use std::{
-    collections::{BTreeMap, HashMap, HashSet, VecDeque},
+    collections::{BTreeMap, HashMap as sHashMap, HashSet, VecDeque},
```

Comment:
The idiomatic way is to state `use revm::HashMap as rHashMap` but perhaps we don't need this and can simply use revm's `HashMap`

Reply:
btw revm's HashMap belongs to alloy_primitives since last update I believe

Reply:
I tried using alloy's `HashMap` everywhere when possible (because foundry now does that, there's no std HashMap) but it ends up clashing with some `zksync` libraries that do use the std HashMap. Imo, aliasing the the standard one is the lesser evil because it is the usage that differes from upstream, this way we have less merge conflicts.

Reply:
Actually I found some more case I could use alloy's HashMap vs std